kubespawner/spawner.py

```diff
@@ -1437,67 +1437,68 @@ def _validate_image_pull_secrets(self, proposal):
-            border-bottom: 1px solid #ccc;
-            padding-bottom: 12px;
-        }
+            /*
+                .profile divs holds two div tags: one for a radio button, and one
+                for the profile's content.
+            */
+            #kubespawner-profiles-list .profile {
+                display: flex;
+                flex-direction: row;
+                font-weight: normal;
+                border-bottom: 1px solid #ccc;
+                padding-bottom: 12px;
+            }
 
-        #kubespawner-profiles-list .profile .check {
-            display: flex;
-            align-content: center;
-            padding: 12px;
-        }
+            #kubespawner-profiles-list .profile .radio {
+                padding: 12px;
+            }
 
-        #kubespawner-profiles-list .profile .option{
-            display: flex;
-            align-items: center;
-            padding-bottom: 12px;
-        }
+            /* .option divs holds a label and a select tag */
+            #kubespawner-profiles-list .profile .option {
+                display: flex;
+                flex-direction: row;
+                align-items: center;
+                padding-bottom: 12px;
+            }
 
-        #kubespawner-profiles-list .profile .option label{
-            font-weight: normal;
-            margin-right: 8px;
-            min-width: 96px;
-        }
+            #kubespawner-profiles-list .profile .option label {
+                font-weight: normal;
+                margin-right: 8px;
+                min-width: 96px;
+            }
         </style>
 
         <div class='form-group' id='kubespawner-profiles-list'>
-        {% for profile in profile_list %}
-        {# Wrap everything in a <label> so clicking anywhere selects the option #}
-        <label for='profile-item-{{ profile.slug }}' class='profile'>
-            <div class='check'>
-                <input type='radio' name='profile' id='profile-item-{{ profile.slug }}' value='{{ profile.slug }}' {% if profile.default %}checked{% endif %} />
-            </div>
-            <div>
-                <h3>{{ profile.display_name }}</h3>
-                {% if profile.description %}
-                <p>{{ profile.description }}</p>
-                {% endif %}
-                {% if profile.options %}
-                <div class='options'>
-                    {% for k, option in profile.options.items() %}
-                    <div class='option'>
-                        <label for='option-{{profile.slug}}-{{k}}'>{{option.display_name}}</label>
-                        <select name="option-{{profile.slug}}-{{k}}" class="form-control">
-                            {% for k, choice in option['choices'].items() %}
-                            <option value="{{ k }}" {% if choice.default %}selected{%endif %}>{{ choice.display_name }}</option>
-                            {% endfor %}
-                        </select>
+            {%- for profile in profile_list %}
```

Comment:
I added whitespace chomping to align with how I'm used to doing it in the helm charts, it doesn't matter in practice though, but the emitted HTML is now free of a lot of blank lines.